fastseq_cli/transformers_generate.py

```diff
@@ -533,6 +574,9 @@ def run_generate():
                         help="The parameter for repetition penalty. 1.0 means no penalty.")
     parser.add_argument("--do_sample", action="store_true",
                         help="Whether or not to use sampling ; use greedy decoding otherwise.")
+    parser.add_argument("--num_return_sequences", type=int,
+                        default=None, required=False, 
```

Comment:
nit: Will it better to set the default value as `1` instead of `None`? `None` seems to be no return sequences.

Reply:
I did it this way for consistency with how the HF params are added. In HF it defaults to None: https://github.com/huggingface/transformers/blob/0aac9ba2dabcf97b22da8e9a0875b52b2113c062/src/transformers/generation_utils.py#L854

Reply:
Got it.